### ISSUE TITLE
CC a11y audit - date picker component

### ISSUE THREAD

**Issue by amycole501** [uswds/uswds#5598]
Audit the date picker component for WCAG compliance and critical checklist checks using three manual accessibility testing methods.
[Critical Checklist spreadsheet](https://docs.google.com/spreadsheets/d/1fAJaObQHv7aIaeUpH4fTLmauhmugMtNCrbpZe4BJcPY/edit?usp=sharing)

This ticket will be considered done once the following content has been written and associated WCAG checks are in place for the [date picker component](https://designsystem.digital.gov/components/date-picker/) component:

WCAG criteria related to date picker component elements

How to perform manual testing (critical checklists) and related user-focused questions for downstream teams to test. 
a. Zoom mag to 200% 
b. Keyboard 
c. JAWS 

### A11Y
- [x] Establish the success criteria we need to check against 
- [x] Perform each check 
- [ ] Review with other a11y team member
- [ ] Reassess (as needed)
- [ ] Submit any failures as bug tickets

### Content and Dev

- [ ] https://github.com/uswds/uswds-site/issues/2430

## References
[Critical Checklist spreadsheet](https://docs.google.com/spreadsheets/d/1fAJaObQHv7aIaeUpH4fTLmauhmugMtNCrbpZe4BJcPY/edit?usp=sharing)
[List of all components](https://docs.google.com/spreadsheets/d/1ASzZ80oI6WVWAifqPO6gYAqrsRaiZs_qGTOStuQPyKs/edit?usp=sharing)

**Comment by amycole501** [comment]
The initial audit is done; It still needs an additional review and finalization before going live on the site. One issue was detected; need to verify with team before putting in an issue.

**Comment by amycole501** [comment]
Second audit is on temporary hold, waiting for some updates to be posted. Will move on to another component for re-audit in current sprint.
